src/storage/src/table/test_relational_table.rs

```diff
@@ -346,6 +346,8 @@ async fn test_state_table_update_insert() -> StorageResult<()> {
         .await
         .unwrap();
     assert_eq!(row1, None);
+
+    epoch += 1;
```

Comment:
```suggestion
```
?

Reply:
Because this test didn't commit two write batches of the same epoch, and will cause sanity check to fail.